### PR TITLE
0.11 to 0.12 migration guide

### PR DESCRIPTION
**Pull request by IceSentry**
This is still very WIP and only contains the original output of the `generate-release` tool.

Feel free to submit a PR to my branch with any fixes.

### CODE REVIEW

File: content/learn/migration-guides/0.11-0.12/_index.md [hunk 1]
```diff
@@ -0,0 +1,827 @@
...
+
+```rust
+// Old
+AssetPath::new("logo.png", None);
+
+// New
+AssetPath::new("logo.png");
+
+// Old
+AssetPath::new("scene.gltf", Some("Mesh0");
```
Comment (OleStrohm) [inline content/learn/migration-guides/0.11-0.12/_index.md:42]:
```suggestion
AssetPath::new("scene.gltf", Some("Mesh0"));
```

File: content/learn/migration-guides/0.8-0.9/_index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 +++
 title = "0.8 to 0.9"
-weight = 4
+weight = 5
```
Comment (rparrett) [inline content/learn/migration-guides/0.8-0.9/_index.md:3]: Oops

Comment (IceSentry) [reply content/learn/migration-guides/0.8-0.9/_index.md:3]:
```suggestion
weight = 4
```
Seems like I messed up the merge commit

**Comment by rparrett** [comment]
Notes (will update as I migrate stuff):

- [ ] `Transparent2d` has a new `dynamic_offset` field. Not sure what to do with it.

**Review by cart — APPROVED** [review]
Thanks a ton for putting this together! I think this is in a reasonable place. We can iterate as needed (even post-release if necessary)

**Pull request merged**